### DISCUSSION TITLE
Who's hiring (Sep 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#40173] [Ideas]
# What is this?
New month, new post! This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #40172.

*If you're looking for last month's discussion, see #39273. At least one company has [hired someone](https://twitter.com/Nutlope/status/1475929984174895105) as a result of it!*

**Comment by guigrpa** [comment]
**Lobelia Earth | Front-End Software Developer | Barcelona (preferred) or remote | Full-time | [Job post](https://www.lobelia.earth/jobs/front-end-software-developer)**

Lobelia is a pioneering company in the use of Earth Observation satellites to address the climate emergency. We develop and operate scientifically-sound services to study the state of the Earth’s resources, and provide key data for adaptation and mitigation measures in face of the climate crisis.

We’re looking for a **Front-End Software Developer** to strengthen our Applications & Software team.

*Why?*

Do you want to develop challenging applications beyond the run-of-the-mill e-commerce site? Maybe you have a deep interest for maps and interactive graphs?

At Lobelia we believe in the importance of designing engaging interfaces to drive climate action. We are passionate about creating new ways to engage users and support their understanding of climate science through interactive and appealing interfaces.

Also, we [love maps](https://myocean.marine.copernicus.eu/-/flhiwwam6a). Do you know (or would like to) the ins & outs of 5 different geographic projections? Curious about the three administrative units that always cause headaches to digital cartographers (yes, those: Fiji, Russia and above all: [Antarctica](https://myocean.marine.copernicus.eu/-/5qxqk8h3yg)? This can definitely be a job for you. We create the fastest, most beautiful maps and graphs to explore climate datasets and satellite imagery. Mapbox and Google Maps didn’t fulfil all of our needs, so we built our own multi-projection mapping engine, TeroMaps, and rendered our own basemaps, Lobelia Oceans Dark/Light and Lobelia Coast. Come and help us reach the next level in **geospatial data exploration**.

*What?*

Main elements of our web stack at the Applications & Software team: d3, Next.js, React, JavaScript/TypeScript, Vercel (+Netlify...), AWS (+GCP, CloudFlare).

But we also work hand-in-hand with the Data Engineering team to build end-to-end solutions including many Python components: xarray, zarr, blosc, dask... How else could we materialise our vision of **full-resolution, Petabyte-scale dataset exploration on the web**?

*How?*

Check out the job post: https://www.lobelia.earth/jobs/front-end-software-developer -- and reach out to us at jobs@lobelia.earth.

**Reply by wiggledom** [reply]
Hello
Thanks for your positing and very interested in that position
I just sent my resume to your email
Please check 
I 'd appreciate it if you give me a chance to work with your company
Thanks

**Comment by jeremydouglas** [comment]
[AgencyAnalytics](http://agencyanalytics.com/) | Web Engineer | Remote | Full-time | [Job Post](https://agencyanalytics.com/company/careers#job-1064449)

We are hiring for the [Web Engineer](https://agencyanalytics.com/company/careers#job-1064449) position to help re-platform our marketing website to Next.js / Contentful and then update with new features and maintain the website going forward. Designs are provided by our UI design team, but it will help to have a combination of an eye for design and sufficient TypeScript / Next.js skill.

You will be working in a small team of three developers to support our marketing website. I have been at this company for 1.5 years and have really enjoyed it. Being a product driven and self-funded company is definitely a breath of fresh air in the SaaS market.

About the company:
AgencyAnalytics is a reporting platform that helps digital agencies automate their client reporting. We have been in business since 2010, are 100% employee-owned, and are growing fast.

Key benefits:
- Fully remote, no location restrictions except being authorized to work in Canada.
- Profit-sharing distributed quarterly
- 4 weeks vacation
- 100% self-funded company

If you are interested in applying and have any questions, feel free to [reach out to me](https://twitter.com/jeremydouglas)

**Reply by smithsoft99** [reply]
Hello, Douglas.
I am Alex from Canada.
I sincerely hope everyone is doing well and safe from covid-19.
I’m a computer nerd at heart. I’ve been fascinated with computers and technology since I was 5 (I’m 31 now) when my dad first introduced me to the Commodore 64 computer. I would sit in front of our TV set and not only play video games (favorites included Shamus, Frogger, Donkey Kong, Ms. Pacman, and the classic text adventure Zork), but I would read the C-Basic manual with my older brother and learn how to make simple programs.

I had a re-introduction to computers at age 12 when I got my first PC (a Packard Bell 286). I was officially hooked on computer to computer communication when a friend and I called our first BBS and thought we had somehow accidentally accessed a secret government mainframe. “WHAT IS YOUR NAME?” It asked. I gave a fake name. “WHAT IS YOUR LAST NAME?” ….I hung up and unplugged the phone line from the wall. I was scared to death at first, but underneath that fear was a boiling excitement that never really left me; that drove me to learn more and more as the years went on.

From there I started tearing computers apart and learning how they worked inside and out. I also immersed myself web development.

So I think I am a full stack developer.

s a full-stack web developer following expertise.
-HTML, HTML5, CSS, CSS3, SCSS, JavaScript, TypeScript, Java, jQuery, JSON, AJAX, PHP
-Bootstrap 3.0, Bootstrap 4.0, Metronic
-Tailwind CSS, MaterialUI, AntDesign, ReactStrap
-React.js, React Native, React/Redux, Redux/Saga, React Hooks, React strap
-Vue.js, Vuex, Vue-cli, Vue
-Python, Django, Golang
-Blockchain, Solidity, NFT, Ethereum, Rust
-Node.js, Express, Next.js, Nest.js, GraphQL
-MySQL, MongoDB, SQLite, PostgreSQL
-Github, GitLab, Bitbucket, SourceTree
-DevOPS, AWS, CI / CD, Versel

And I have full and extensive experience in
-Pixel perfection and fully responsive across all devices, PSD to HTML
-Datatables with Bootstrap, React, and Vue
-Development of Reusable React, Vue Component, and Directive
-Firebase, MongoDB, MySQL programming
-Database Design
-Backend Restful API

Almost everyone carries an Internet capable device in their pocket. Just think about that for a minute or two, at the lowest level possible – a worldwide, high bandwidth network that a very large portion of the general population has access to. The possibilities are endless in thinking of the next killer app for this incredible technology available to us all today that could benefit people all over the planet (and soon to be other planets, hopefully! 😉 )

I love where the growing power, capacity and capability of the Internet is headed.
If anyone wants my help, please ping me anytime.

Location: Toronto, ON, Canada
Remote: Yes

LinkedIn profile: https://www.linkedin.com/in/alex-smith-717499249/
Email: [alexsmithsuperstar1004@gmail.com](mailto:alexsmithsuperstar1004@gmail.com)
Thank you.

**Reply by jeremydouglas** [reply]
Thanks @smithsoft99, please use the form linked at the bottom of [this page](https://agencyanalytics.com/company/careers#job-1064449) to apply.

**Comment by dwnnoutnsd** [comment]
Repod | Full Stack Engineer | Remote | Full-time/Part-time | https://repod.io

Hey I’m Travis, CEO of Repod, an all-in-one podcast app/community platform that helps podcasters to turn their listeners into a community. Repod is like Substack/Patreon, but for podcasts.

We're a young company that recently raised a pre-seed round. We are looking for a full stack engineer that will take on a CTO/cofounder type of role. The ideal person is someone who’s familiar with building, iterating, and launching products of their own. Repod started as a mobile iOS/Android app but your role will primarily be working on a web version of Repod.

Tech stack:

Web: Next.js + Redux

iOS/Android podcast app : React Native, Firebase, Typescript, Swift/Java (we have a native audio player)

Apply by getting in touch with me:

Email: travis [@] repod.io

Twitter: https://twitter.com/travisosterhaus (DM me)

**Comment by leerob** [comment]
Related: https://twitter.com/Suhail/status/1567260126431842304

> Looking for one more front end founder/hacker  that knows React/TS/Postgres and is in between things. Should have hacked on front-end things for 4+ years.
> 
> US/Canada only — remote ok

**Comment by rmanalan** [comment]
**Arcade | Full Stack Engineer | Remote (US/Canada) | Full-time | https://arcade.software/**

👋🏽 Hey! I'm Rich, CTO @ Arcade. Arcade 🕹️ is a service that makes it super easy to create interactive demos. Check out our [showcase](https://www.arcade.software/showcase) for examples of our customers and how they use our product. We're an early stage VC funded company -- a little over a year into our journey... so you'd be joining a small (7) person team of amazing people. Here's [the full job description](https://arcade-software.notion.site/Full-Stack-Engineer-1f8a36ff8c6649299c1d22a76dddec99).

Our stack primarily consists of: NextJS, TypeScript, Firebase/GCP.

If you're interested, DM me [here](https://twitter.com/rmanalan) or [here](https://www.linkedin.com/in/manalang/).

**Comment by Nutlope** [comment]
**Vercel | Engineering Manager, Turborepo | Remote (US, Canada, Europe, or Asia) | Full-time | [Link](https://vercel.com/careers/engineering-manager-turborepo-amer-4531844004)**

Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.

Feel free to apply using the link above.

**Comment by rcronin** [comment]
Hey there everyone!

Dayton Freight is currently looking for an experienced Next.js developer to help us kick off an enterprise modernization effort. In this role you will be instrumental in developing a large scale enterprise-wide Multi Zone Next.js application.

Full time position / 100% remote / competitive benefits.

Please messaged me on [LinkedIn](https://www.linkedin.com/in/ryan-cronin-71032328/) for more information!

**Reply by christoph724** [reply]
I want to know your email ID.
Please reply to me.
Thanks.

**Comment by eugene-luzgin** [comment]
Dear.

As a senior web developer with 8+ years of web development experience, I provide you accomplish your goals in the most efficient way!

Also I have rich experience with Front and Backend-end web application built in React, Vue and Node Express, PHP, Laravel, CI etc.

I Implemented a lot of sites with my specific skills using React, Redux, Context, React Hook, Sass, CSS library like Material & Ant Design & Tailwind and deliver pixel perfect User Interface you want at time

During that period, I have developed a lot of project including Enterprise software (ERP & CRM), E-commerce & Retail, Real Estate & Property and so on.

I believe that "Success is no accident" and work hard and concentrate on being honest.
I've good analytical, communication, teamwork, time management skills, and problem-solving abilities.
Also I used to work with AWS, MySQL, MongoDB, Jenkins, Docker, Kubernetes, too.
I can work 40 hours per week.


Backend:
- Node.js, python, Django, Ruby on Rails, Laravel, Spring-boot, CodeIgniter, PHP, .Net
- MySQL, MSSQL, PostgreSQL, MongoDB
Frontend:
- React, Vue, Angular, Graphql
- Bootstrap, Semantic UI, Material UI, Metro UI, UIKit

Work with the Agile methodology and have experience with the following tools JIRA, Trello and GitHub. Feel free to contact me for your project


Thanks & regards

**Comment by sdgandhi** [comment]
Spacetime | Software Engineer (front-end) | Remote | Full-time | negotiable salary | [Spacetime careers link](https://spacetime.xyz/careers/f93ca44d-caef-4a36-841b-b2f2a55ee1c6)

👋  Hey, Sid from Spacetime here. I'm super excited to meet some NextJS engineers!

## About Spacetime

Spacetime is working on a once-in-a-cycle opportunity to build the database layer for web3. What we build and how we build it will have implications for decades. It’s a big vision, but we are laser focused on delivering value to customers today. Balancing long-term vision and concrete delivery is key to our success, and hopefully yours too.

## About the role

This multifaceted position and will build many user-facing and internal parts of Spacetime. It combines full-stack development with UX and DX (developer experience).

This role reports to the CEO.

## Your traits

- You are confident developing in React and NodeJS. You have shipped multiple production products using these technologies.
- You have an eye for layout and design. You can implement projects with limited design direction.
- You can work on multiple projects at one time, balancing speed and quality tradeoffs.
- Bonus: previous experience in web3

## Your area of ownership

Your areas of ownership will include the Explorer (admin console where developers manage databases), example applications, marketing website, automations, integrations and internal tools.

**Reply by wiggledom** [reply]
Hello
Thanks for your posting looking for Next.js and Node.js dev
I have enough experience with react.js and Next.js, I already developed lots of projects using react and Next.js
Also I have rich experience with backend development using node.js.
Briefly speaking, I developed several apis using node.js
I 'd appreciate it if you give me a chance to work with you
Thanks

Best regards

**Reply by intelliseawolf** [reply]
Hello.
Thanks for posting valuable job opportunity. I have a concrete experiences in web development using modern JavaScript and TypeScript frameworks especially in React, Next.js and Node.js.
You can check my recent work in this repo.
https://github.com/intelliseawolf/post-express-next
I am looking forward to discuss the opportunity.
Thanks.

Gmail: intelliseawolf103@gmail.com

**Reply by adeniyii** [reply]
Hey @sdgandhi , is it fine to include you as a referral in the application? Thanks

**Reply by sdgandhi** [reply]
Just a heads up to those replying here, please use the application link: https://spacetime.xyz/careers/f93ca44d-caef-4a36-841b-b2f2a55ee1c6

**Comment by inveve** [comment]
BASK HEALTH | SENIOR ENGINEER | New York, NY (Open to Remote) | Full-time | $100-150k | [LINK](https://www.linkedin.com/jobs/view/3276600028/)

Bask is looking for a leader, a creative, a entrepreneurial developer that can take software from 0 to 1. Bask is building the Shopify for E-Prescribing, a all in one platform for digital health. 

This unique opportunity allows you to make an impact across the company, at an early stage. With high value founding options packages that allow you to really reap the benefits of your work. We are 10 months into development, and it is time we had someone senior join us. 

If you think you have what it takes, but might not be at that senior level yet, don't hesitate to still reach out and we might be able to find a place for you with us. 

Feel free to apply using the link above or reach out to me directly. Thanks!

**Reply by inveve** [reply]
Feel free to submit an application directly to us using this link: https://forms.clickup.com/14211656/f/dhpj8-1864/HSLPXIY59WU5MSCYOO

**Comment by IraES0116** [comment]
EQUUS SOFTWARE| SOFTWARE ENGINEER (FRONT END) | LISBON | Full-time | https://apply.workable.com/equus-software/j/32DB727635/

As a Software Engineer, you will be building out key parts of our framework
that is used to build new products.

You will work with self-organizing development teams to improve the team,
process, product and user experience. Day-to-day, you will collaborate with
your team of developers, business analysts and product owners. You will also
work with subject matter experts from other teams as well as
implementation, support staff and sometimes Clients to produce a steady
stream of new features

Requirements
WHAT YOU BRING
3+ years proficiency with at least one object-oriented programming language (C#, Java, etc.)
Experience with Javascript SPA like Angular/React, Angular 2, TypeScript, JQUERY, Twitter Bootstrap, JavaScript, CSS, HTML
Understanding of complex concepts, such as recursion, reflection, multi-threaded processes, memory management
Proven record of leading and successful delivery of multiple projects
Experience working in both individual and team environments
NICE TO HAVE
4+ years of experience operating in an agile environment (Scrum or Kanban)
Experience with ASP.NET MVC
Experience with Blazor
Test Driven Development using a testing framework like MSTest or NUnit
Experience using an ORM framework like NHibernate or Entity
Framework
Object-oriented design
Database design
T-SQL
Code and database performance analysis
Secure coding practices
Troubleshooting complex problems
Visual Studio
SQL Server Management Studio
UI design
Benefits
Permanent Role – You will not start over when moving projects. We only hire passionate people who are looking for a long-term opportunity with a growing company.
We LOVE your ideas! Innovation Day, once a quarter you can share your fresh ideas.
Diversity/Inclusion - With employees from far-reaching countries across the globe, we've welcome unique mindsets and different backgrounds.
Culture – In Equus the work life balance is valued. For that reason, we try to organize team outings and team buildings activities regularly.
Salary & Benefits - We have highly competitive salaries with an exceptional benefit package, that includes medical coverage, PTO, flexible work hours, plus more.
Great people, great leaders. We only hire the BEST in the market so they can become the best professionals of industry! Our people are always available to help one another.
Yearly team events where you’ll be able to meet all of your colleagues face to face in one place.

**Comment by smithsoft99** [comment]
Hi,
Thank you for your message.
First of all, I sincerely hope you are doing well and safe from covid-19.
I am very interested in this opportunity.
This is my resume.
I look forward to hearing from you soon.
If I missed your call, please send me a message via email.
Warm regards.
Alex
